.appveyor.yml

```diff
@@ -3,7 +3,7 @@ skip_commits:
 
 environment:
 
-  CONDA_INSTALL_LOCN: "C:\\Miniconda36-x64"
+  CONDA_INSTALL_LOCN: "C:\\Miniconda3-x64"
```

Comment:
Please don't. https://github.com/conda/conda/issues/7144

Reply:
Hmmm... OK.  I was probably chasing a red herring with the conda exe thing.  I will revert this.  Thanks for posting the issue.

Reply:
Given recent issues this seems to solve, are we ok leaving `Miniconda36` here short term? This may already be agreeable at this stage. Just trying to confirm.